container/rkt/helpers.go

```diff
@@ -80,7 +123,7 @@ func getRootFs(root string, parsed *parsedName) string {
 
 	bytes, err := ioutil.ReadFile(tree)
 	if err != nil {
-		glog.Infof("ReadFile failed, couldn't read %v to get upper dir: %v", tree, err)
+		glog.Warningf("ReadFile failed, couldn't read %v to get upper dir: %v", tree, err)
```

Comment:
Should this be an error? Are there any circumstances where this would be expected?